www/video/visualProgress.inc.php

```diff
@@ -157,6 +157,8 @@ function GetVisualProgressForStep($localPaths, $runCompleted, $options = null, $
     foreach($frames['frames'] as $time => &$frame) {
       if ($frame['progress'] > 0 && !array_key_exists('startRender', $frames))
         $frames['startRender'] = $time;
+      if (!$frames['visualComplete85'] && $frame['progress'] >= 85)

```

Comment:
Since visualComplete85 isn't initialized, can you change this to:
```
if (!isset($frames['visualComplete85']) && $frame['progress'] >= 85)
```
(or initialize it to zero like visualComplete above but I like the idea that it won't be set if it wasn't calculated)